src/components/Beacon.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import { Helmet } from 'react-helmet'
+
+const Beacon: React.FC = () => (
+    <div>
+        <h1 hidden>{process.env.REACT_APP_HELPSCOUT_BEACON_ID}</h1>
```

Comment:
Is there a better way to insert the beacon id rather than using a h1? Could you directly include it in the script tag? this feels a little weird to me